### PR TITLE
SDK: Replace sdk with deploymentWallet

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-sdk/test/e2e/DeployedWallet.ts [hunk 1]
```diff
@@ -62,4 +62,31 @@ describe('E2E: DeployedWallet', async () => {
...
+
+    it('initial mockToken balance', async () => {
+      const balance = await deployedWallet.getBalance(mockToken.address);
+      expect(balance).to.equal(initialMockTokenBalance);
+    });
+
+    it('initial mockToken balance + sent value', async () => {
+      await mockToken.transfer(deployedWallet.contractAddress, constants.WeiPerEther);
+      const balance = await deployedWallet.getBalance(mockToken.address);
+      expect(balance).to.equal(constants.WeiPerEther.add(initialMockTokenBalance));
```
Comment (cehali) [inline universal-login-sdk/test/e2e/DeployedWallet.ts]: I would just make it more explicit: `initialMockTokenBalance.add(constants.WeiPerEther)`

File: universal-login-sdk/test/e2e/DeployedWallet.ts [hunk 2]
```diff
@@ -62,4 +62,31 @@ describe('E2E: DeployedWallet', async () => {
...
+
+    it('initial eth balance', async () => {
+      const balance = await deployedWallet.getBalance(ETHER_NATIVE_TOKEN.address);
+      expect(balance).to.equal(initialEthBalance);
+    });
+
+    it('initialEthBalance + 1 ETH', async () => {
+      await otherWallet.sendTransaction({to: deployedWallet.contractAddress, value: constants.WeiPerEther});
+      const balance = await deployedWallet.getBalance(ETHER_NATIVE_TOKEN.address);
+      expect(balance).to.equal(constants.WeiPerEther.add(initialEthBalance));
```
Comment (cehali) [inline universal-login-sdk/test/e2e/DeployedWallet.ts]: I would just make it more explicitliy readable: `initialMockTokenBalance.add(constants.WeiPerEther)`

Comment (xitronix) [reply universal-login-sdk/test/e2e/DeployedWallet.ts]: Done

File: universal-login-sdk/lib/api/sdk.ts [hunk 3]
```diff
@@ -197,6 +197,10 @@ class UniversalLoginSDK {
     return contract.lastNonce();
   }
 
+  async getBalance(contractAddress: string, tokenAddress: string) {
+    return this.balanceChecker.getBalance(contractAddress, tokenAddress);
+  }
```
Comment (marekkirejczyk) [inline universal-login-sdk/lib/api/sdk.ts]: No need to add it here, we will deprecate it soon.

**Pull request merged**